site/content/docs/latest/tutorials/kubeapps-on-tce/05-Managing-applications.md

```diff
@@ -0,0 +1,10 @@
+# Managing applications with Kubeapps
+
+You can explore the excellent existing documentation for operating Kubeapps and all its functionalities.
```

Comment:
Even if we were so proud of our docs, we should add these kinds of adjectives in a formal documentation 😝 

```suggestion
You can explore the existing documentation for operating Kubeapps and all its functionalities.
```